producers-commissioning/build.gradle

```diff
@@ -25,3 +37,54 @@ configurations {
   testE2EImplementation.extendsFrom testImplementation
   testE2ERuntimeOnly.extendsFrom runtimeOnly
 }
+
+tasks.register('printSourceSetInformation') {
+  doLast{
+    sourceSets.each { srcSet ->
+      println "["+srcSet.name+"]"
+      print "-->Source directories: "+srcSet.allJava.srcDirs+"\n"
+      print "-->Output directories: "+srcSet.output.classesDirs.files+"\n"
+      print "-->Compile classpath:\n"
+      srcSet.compileClasspath.files.each {
+        print "  "+it.path+"\n"
+      }
+      println ""
+    }
+  }
+}
```

Comment:
This task looks like one that was used in development to check which the source sets were being used.  This can probably be deleted.